models/ac_generator.h

```diff
@@ -46,7 +46,7 @@ ac_generator
 Description
 +++++++++++
 
-This device produces an ac-current sent by a CurrentEvent. The
+This device produces an AC current sent by a CurrentEvent. The
```

Comment:
```suggestion
This device produces AC input sent by CurrentEvents. The
```

Reasoning: AC already has a 'current' in it, cf. [PIN number, LCD display, ...](https://en.wikipedia.org/wiki/RAS_syndrome)